autolens/pipeline/phase/abstract/result.py

```diff
@@ -38,7 +38,10 @@ def source_plane_light_profile_centres(self) -> grids.Coordinates:
 
         These centres are used by automatic position updating to determine the best-fit lens model's image-plane
         multiple-image positions."""
-        return self.most_likely_tracer.source_plane.light_profile_centres
+        try:
+            return self.most_likely_tracer.source_plane.light_profile_centres
+        except:
```

Comment:
Broad, unhandled, unlogged exceptions are scary.

If you're not going to log it then make it a specific exception